products/BellHybrid/apps/application-bell-meditation-timer/windows/StatisticsWindow.hpp

```diff
@@ -1,11 +1,12 @@
-// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #pragma once
 
 #include "Contract.hpp"
 
 #include <apps-common/windows/AppWindow.hpp>
+#include <ListView.hpp>
```

Comment:
can be forward declared

Reply:
It was added only for test purposes and will be replaced with the different list view mechanism in the future PRs. That's why I didn't bother to properly forward declare it. Nevertheless, fixed:)